package.json

```diff
@@ -41,7 +41,7 @@
     "npm-run-all": "~4.1.5",
     "prom-client": "14.2.0",
     "request": "^2.81.0",
-    "scubaclient": "git+https://github.com/scality/scubaclient.git",
+    "scubaclient": "git+https://github.com/scality/scubaclient.git#fb7375a9298bda7df0e9f9ed81d7fc5b363590a9",
```

Comment:
To be changed to proper version tag before merge I guess ?

Reply:
We do not have any released tag yet, for scubaclient